src/context/AlertContext.tsx

```diff
@@ -0,0 +1,11 @@
+import { createContext } from 'react';
+
+type AlertContextValues = {
+  showAlert: boolean;
+  setShowAlert: Dispatch<SetStateAction<boolean>>;
+};
+
+export const AlertContext: Context<AlertContextValues> = createContext<AlertContextValues>({
+  showAlert: false,
+  setShowAlert: () => {},
```

Comment:
the variable names a bit confusing, how about something like:
```suggestion
  alertShown: false,
  setAlertShown: () => {},
```

Reply:
updated in https://github.com/RR-Ardhika/murojaah/pull/7/commits/7e3a825fb939daf367835007e1506ec2381dffcb, thank you